File: tuts/010-cloudmap-service-discovery/swift/Sources/GettingStartedScenario.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSCloudMap

@main
struct GettingStartedScenario {
    static func main() async throws {
        let client = try CloudMapClient(region: "us-east-1")
        let wrapper = CloudMapWrapper(client: client)
        print("Running CloudMap getting started scenario...")
        // TODO: setup, interact, teardown
        let _ = wrapper
    }
}

File: tuts/004-cloudmap-custom-attributes/php/GettingStartedScenario.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

require 'vendor/autoload.php';

use Aws\CloudMap\CloudMapClient;

$client = new CloudMapClient(['region' => 'us-east-1']);
echo "Running CloudMap getting started scenario...\n";
// TODO: setup, interact, teardown
echo "Scenario complete.\n";
